packages/html/src/expansion-panel/expansion-panel.spec.tsx

```diff
@@ -8,7 +8,7 @@ const states = [
     States.hover,
     States.focus,
     States.disabled,
-    'expanded'
+    'expanded' as const
```

Comment:
In my opinion we should not add _expanded_ state here, as it leads to **collapsed + expanded** state, which is not correct. Expanded / collapsed are more like "logical" states, while the others are "interactive"